forms-flow-api-utils/src/formsflow_api_utils/exceptions/__init__.py

```diff
@@ -110,32 +112,44 @@ def handle_sqlalchemy_error(e, model=None):
     raise BusinessException(error_message, code=error_code, details=details)
 
 
-def register_error_handlers(api):
+def register_error_handlers(api, sentry_capture_handled_errors: bool = False):
```

Comment:
where will get the value to this variable : `sentry_capture_handled_errors ` , from env ?

Reply:
Yes it will be used from env config for the containers (api). As this is utils, it won't read from env, instead gets passed as argument on app initialization from api